{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -12,9 +12,8 @@ maintainers = [
   { name="Simon J.L. Billinge group", email="simon.billinge@gmail.com" },
 ]
 description = "{{ cookiecutter.project_short_description }}"
-{%- set keywds = [cookiecutter.keyword_1, cookiecutter.keyword_2, cookiecutter.keyword_3] -%}
-{% set keyword_list = keywds | select('string') | map('trim') | reject('equalto', '') | list %}
-keywords = {{ keyword_list }}
+{%- set keywords_list = (cookiecutter.keywords.split(',') if cookiecutter.keywords.strip() else []) %}
+keywords = {{ keywords_list }}
 readme = "README.rst"
 requires-python = ">=3.9"
```

Comment:
This should be based on the minimum python version as well

Reply:
You mean the requires-python variable?

Reply:
Yes, also is it possible to do the version splitting by finding the period rather than by length?

Reply:
I think it might be, I'll look into it quickly and push a change if it works

Reply:
Yep, it worked. Commit should be pushed now.

Reply:
Nice